metrics/lib/features/dashboard/domain/entities/metrics/build_result_metric.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/foundation.dart';
+import 'package:metrics/features/dashboard/domain/entities/core/build.dart';
+
+/// Represents the build results metric entity.
+@immutable
+class BuildResultMetric {
+  final List<BuildResult> buildResults;
+
+  /// Creates the [BuildResultMetric].
+  ///
+  /// [buildResults] represents the results of several builds.
+  const BuildResultMetric({this.buildResults = const []});
+}
+
+/// Represents the CI build [result] on specified [date].
+///
+/// Contains the data about build [url] and build [duration].
+@immutable
+class BuildResult {
```

Comment:
I know we discussed it a bit to leave such classes in the same file but from a review standpoint, it looks that it might be better to have separate files. What do you think?